docs_src/microservices/configuration/Ch-Configuration.md

```diff
@@ -158,16 +186,26 @@ for edgex-core-data are:
 
 -   /edgex/core/1.0/edgex-core-data/Clients
 -   /edgex/core/1.0/edgex-core-data/Databases
--   /edgex/core /1.0/edgex-core-data/Logging
+-   /edgex/core/1.0/edgex-core-data/Logging
 -   /edgex/core/1.0/edgex-core-data/MessageQueue
 -   /edgex/core/1.0/edgex-core-data/Registry
+-   /edgex/core/1.0/edgex-core-data/SecretStore
 -   /edgex/core/1.0/edgex-core-data/Service
 -   /edgex/core/1.0/edgex-core-data/Writable
 
 Any configuration settings found in the `Writable` section shown above
 may be changed and affect a service's behavior without a restart. Any
 modifications to the other settings would require a restart.
 
+*NOTE: As of the Geneva release, the support-logging service is deprecated. In the Readable section of each service's configuration there is currently a Logging section shown below. The recommendation is that you not change the default values in this section unless you have a specific reason for doing so.*
+
+```
+# Remote and file logging disabled so only stdout logging is used
+[Logging]  
+EnableRemote = false  
+File = ''
+```
+
 ## Registry
```

Comment:
This section needs to be reworked for the separation of registry and configuration providers. Suggest titling this as `Registry provider` and add a new `Configuration provider` section